src/index.ts

```diff
@@ -234,6 +249,20 @@ class MultinetAPI {
     return this.client.delete(`/workspaces/${workspace}/tables/${table}`);
   }
 
+  public tableMetadata(workspace: string, table: string): Promise<TableMetadata> {
+    return this.client.get(`/workspaces/${workspace}/tables/${table}/metadata`);
+  }
+
+  public async tableColumnTypes(workspace: string, table: string): Promise<ColumnTypes> {
+    const metadata = await this.tableMetadata(workspace, table);
+
+    const types: ColumnTypes = {};
+    metadata.table.columns.forEach((entry) => {
+      types[entry.key] = entry.type;
+    });
```

Comment:
Perhaps we can change the structure of the underlying data to match this in the future, so this transform isn't necessary.

Reply:
Oh yeah, that's something to think about. I feel like in general, lists of data are "better" than dicts of data because ordering is explicit... but maybe that just doesn't matter here.

Opened #24 to track this.